book/03-git-branching/sections/rebasing.asc

```diff
@@ -1,28 +1,24 @@
 [[_rebasing]]
-=== Rebasing
+=== Pag-rebase
 
-(((rebasing)))
-In Git, there are two main ways to integrate changes from one branch into another: the `merge` and the `rebase`.
-In this section you'll learn what rebasing is, how to do it, why it's a pretty amazing tool, and in what cases you won't want to use it.
+(((pag-rebase)))
+Sa Git, mayroong dalawang pangunahing mga paraan upang pagsamahin ang mga pagbabago mula sa isang branch patungo sa iba: ang `merge` at ang `rebase`. Sa seksyong ito matututo ka kung ano ang pag-rebase, paano ito gawin, bakit ito ay isang medyo kahanga-hangang kasangkapan, at sa anong kaso mo hindi gugustuhing gamitin ito.
 
-==== The Basic Rebase
+==== Ang Batayan ng Rebase
```

Comment:
I think it first more Ang Pangunahing Rebase   since batayan is basis

Reply:
No other issues found. Ready for merging . Thank you for contribution

Reply:
Thank you for your inputs @robin-ho and @Cyyy1998. Will update my contribution basing on @robin-ho's review.